lib/assets.js

```diff
@@ -69,9 +69,10 @@ TradeOfferManager.prototype._addDescriptions = function(items, callback) {
 	this._requestDescriptions(descriptionRequired, (err) => {
 		if (err) {
 			callback(err);
-		} else {
-			callback(null, this._mapItemsToDescriptions(null, null, items));
+			return;
```

Comment:
For what it's worth, while `return callback()` is more compact, I dislike it as it makes my IDE yell at me for only returning a value sometimes.